src/popup/components/TransactionDetailsBase.vue

```diff
@@ -258,42 +171,27 @@ export default defineComponent({
   components: {
     PayloadDetails,
     TransactionErrorStatus,
-    TransactionDetailsPoolTokens,
-    TransactionTokens,
     TransactionOverview,
     TokenAmount,
     DetailsItem,
     LinkButton,
     CopyText,
-    SwapRoute,
-    SwapRates,
     AnimatedPending,
     ExternalLink,
-    Avatar,
-    DialogBox,
   },
   props: {
     transaction: { type: Object as PropType<ITransaction | undefined>, required: true },
     coinSymbol: { type: String, required: true },
     transactionFee: { type: Number, required: true },
-    isSwap: { type: Boolean },
-    isPool: { type: Boolean },
     tokenSymbol: { type: String, required: true },
     totalAmount: { type: Number, required: true },
-    isErrorTransaction: { type: Boolean },
-    isDexAllowance: { type: Boolean },
-    isDex: { type: Boolean },
+    isErrorTransaction: Boolean,
+    showHeader: Boolean,
+    showAmount: { type: Boolean, default: true },
```

Comment:
I strongly recommend to not use the bool props with the default value of `true`. If you want to display something by default I suggest to inverse the name, e.g.: `hideAmount`.